students/karl_perez/lesson06/mailroom4.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+#Mailroom part 3
+import sys  
+
+#donors
+donors = [["Krystal Perez", 50.00, 250.00],
+    ["Eddie Lau", 32.50],
+    ["Jimmy Jack", 200.00, 350.00, 400.00],
+    ["Grace Cool", 120.00, 75.00],
+    ["Adriel Molina", 45.00, 450.00]]
```

Comment:
this should continue to use dictionary as the main data structure as that it the ideal structure here